src/py42/_internal/client_factories.py

```diff
@@ -114,6 +117,20 @@ def get_detection_list_user_client(self, user_client):
             )
         return self._detection_list_user_client
 
+    def get_alert_rules_client(self):
+        if not self._alert_rules_client:
+            session = self._get_jwt_session(u"FedService-API_URL")
```

Comment:
Need to know exact string in place of `u"FedService-API_URL"`.  Tried to find it through `_get_sts_base_url` but it didn't help.